<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabInit">
       <attribute name="title">
        <string>Init</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTextBrowser" name="tbInit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabManager">
       <attribute name="title">
        <string>Manager</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="lModems">
          <property name="text">
           <string>Modems...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bPowerOnModem">
          <property name="text">
           <string>Power on modem</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bOnlineModem">
          <property name="text">
           <string>Online modem</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabNetwork">
       <attribute name="title">
        <string>Network</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lNetwork">
          <property name="text">
           <string>Properties...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="tbNetRegPropertyChange"/>
        </item>
        <item>
         <widget class="QPushButton" name="bRegister">
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bGetOperators">
          <property name="text">
           <string>Get operators</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bScan">
          <property name="text">
           <string>Scan</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGsmCall">
       <attribute name="title">
        <string>GSM call</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="tbCallNum">
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string>800123456</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bCall">
            <property name="text">
             <string>Dial</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="tbCallId">
            <property name="text">
             <string>Call ID:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bRelease">
            <property name="text">
             <string>Release</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bActivateConference">
            <property name="text">
             <string>Activate
Conference</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bActivateCall">
            <property name="text">
             <string>Activate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="bReleaseAll">
            <property name="text">
             <string>Release
All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bReleaseHeld">
            <property name="text">
             <string>Release
Held</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bJoin">
            <property name="text">
             <string>Join</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bHoldActive">
            <property name="text">
             <string>Hold active</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLineEdit" name="tbTransferNumber">
            <property name="text">
             <string>608828973</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bTransfer">
            <property name="text">
             <string>Transfer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lLastCall">
          <property name="text">
           <string>Last call:</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bListCalls">
          <property name="text">
           <string>List calls</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGsmSms">
       <attribute name="title">
        <string>SMS</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QCheckBox" name="cbReport">
            <property name="text">
             <string>Report</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bSend">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lSmsSplit">
          <property name="text">
           <string>Split info...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Phone number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tbSmsPhoneNumber"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="tbSmsContent"/>
        </item>
        <item>
         <widget class="QPushButton" name="bRetrieveTextMessages">
          <property name="text">
           <string>Retrieve text messages</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSim">
       <attribute name="title">
        <string>SIM</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QPushButton" name="bSimInfo">
          <property name="text">
           <string>Sim info</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPdp">
       <attribute name="title">
        <string>PDP</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>APN:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tbApn">
          <property name="text">
           <string>internet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tbUsername"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tbPassword"/>
        </item>
        <item>
         <widget class="QPushButton" name="bSetCredentials">
          <property name="text">
           <string>Set credentials</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tbActivateContext">
          <property name="text">
           <string>Activate context</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bDeactivateContext">
          <property name="text">
           <string>Deactivate context</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPim">
       <attribute name="title">
        <string>Pim</string>
       </attribute>
       <widget class="QPushButton" name="bQueryMessages">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>311</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Query messages</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabLog">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="tbLog"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
